es/es-server/src/main/java/org/elasticsearch/snapshots/SnapshotsService.java

```diff
@@ -113,6 +115,8 @@
 public class SnapshotsService extends AbstractLifecycleComponent implements ClusterStateApplier {
     private static final Logger logger = LogManager.getLogger(SnapshotsService.class);
 
+    public static final Version NO_REPO_INITIALIZE_VERSION = Version.CURRENT;
```

Comment:
Not sure what is the right version to use here, in the original code it is `Version.V_8_0_0`

Reply:
```suggestion
    public static final Version NO_REPO_INITIALIZE_VERSION = Version.V_4_1_0;
```


`Version.CURRENT` shouldn't be used to decide when an internal feature first became available because it will change with each new feature version. Decisions like that must be frozen.